Britannia/localization/english/event_localization/BTWK_events_l_english.yml

```diff
@@ -1,25 +1,50 @@
-  BTWK_frontier_event.0201.b_imperial:0 "Destroying the wall may placate the Northern Britons."  # Unsure why imperials would destroy the wall, other than to improve relations with local celts.
-  BTWK_frontier_event.0201.b:0 "We shall destroy the wall for materials."
-  BTWK_frontier_event.0201.c:0 "I will decide at another time."
-  BTWK_frontier_event.0201.c_tt:0 "#I The decision may be taken again at a later date. #!"
\ No newline at end of file
+  BTWK_frontier_event.0201:0 "The End of Hadrian's Wall"  # ROOT of decision trigger
+  BTWK_frontier_event.0201_desc:0 "The Rome of old has been gone from Britannia for many years. Behind it left great feats of engineering that have been left to crumble and decay, much like its successor rump states on the Isle. One of such constructions was #italic Hadrian's Wall#!. Constructed by Emporer Hadrian, it stood definitively as the northern-most border of Rome, designed to keep control of the Caledonians. Without Rome, it served, and still serves, no purpose. There was no border to defend - the land split between warring factions. \n\nAs such, denizens of the region around the wall had begun to repurpose the old stone to improve the infrastructure of their settlements, given it was so easily accessible. We will follow in their footsteps, and use its stone to prompt the development of the surrounding lands."
+  BTWK_frontier_event.0201.a:0 "Out with the old..."
+  BTWK_frontier_event.0201.a_tt:0 "All instances of the [special_buildings|E] #italic Hadrian's Wall#! or #italic Hadrian's Wall - Restored#! will be ruined."
+
+  BTWK_frontier_event.0202:0 "The End of Hadrian's Wall"  # Every other player
+  BTWK_frontier_event.0202_desc_header:0 "[GetScope(destroyed_hadrians_wall_char).Char.GetName] has ordered the demolition of Hadrian's Wall"
+  BTWK_frontier_event.0202_desc_header_vassal:0 "Our liege, [GetScope(destroyed_hadrians_wall_char).Char.GetName], has ordered the demolition of Hadrian's Wall."
+  BTWK_frontier_event.0202_desc_main:0 "\n\n [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] hopes that now the wall no longer serves any meaningful purpose, its stone could be repurposed for the developing settlements throughout [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] realm."
+  BTWK_frontier_event.0202_desc_imperial_like_destroyer:0 "[GetScope(destroyed_hadrians_wall_char).Char.GetFirstName] has done this despite [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] his heritage, from which is intrinsically linked to the momument. For this act, many of his kinsmen have been severely angered."
+  BTWK_frontier_event.0202_desc_else_destroyer:0 "Having no ancient ties to the land, and the wall which sits atop it, [GetScope(destroyed_hadrians_wall_char).Char.GetFirstNamePossessiveNoTooltipRegnal] decision was  tenable. The act has angered the likes of the Britons and of the Latins, who have ancient ties to the monument."
```

Comment:
Having no ancient ties to the land, and the wall which sits atop it, [GetScope(destroyed_hadrians_wall_char).Char.GetFirstNamePossessiveNoTooltipRegnal] decision was  tenable. The act has angered the likes of the Britons and of the Latins, who have ancient ties to the monument. -> While [ruler name] seems to have regarded this as purely a practical move, many Britons and Latins, are furious at what they see as a callous disregard for the island's Roman heritage.

Reply:
Much better than what I had, lol